tests/vision_encoder_decoder/test_modeling_tf_vision_encoder_decoder.py

```diff
@@ -321,7 +324,7 @@ def check_pt_tf_equivalence(self, pt_model, tf_model, inputs_dict):
         self.assertEqual(len(tf_outputs), len(pt_outputs), "Output lengths differ between TF and PyTorch")
 
         for tf_output, pt_output in zip(tf_outputs, pt_outputs):
-            self.assert_almost_equals(tf_output.numpy(), pt_output.numpy(), 1e-3)
+            self.assert_almost_equals(tf_output.numpy(), pt_output.detach().to("cpu").numpy(), 1e-3)
```

Comment:
note that `detach()` is not needed here since it's wrapped in a `torch.no_grad()` but doesn't really matter so ok for me